File: Solutions/Python/Fitting_The_Array.py
class Solution:
    def isFit (self,arr, brr, n) : 
        #Complete the function
        arr = sorted(arr)
        brr = sorted(brr)
        res = 0
        for i in range(n):
            if arr[i]<=brr[i]:
                res+=1
        if res==n:
            return True
        else:
            return False

File: Solutions/Java/Fitting_The_Array.java
class Solution{
    // Function for finding maximum and value pair
    public static boolean isFit (int arr[], int brr[], int n) {
        //Complete the function
        Arrays.sort(arr);
        Arrays.sort(brr);
        boolean ans = true;
        for(int i=0; i<n; i++) {
            if (arr[i] <= brr[i]) {
                continue;
            }
            else {
                ans = false;
                break;
            }
        }
        return ans;
    }
}
